src/gridDynMain/CMakeLists.txt

```diff
@@ -8,30 +8,30 @@
 # For details, see the LICENSE file.
 # LLNS Copyright End
 
-set(griddynMain_sources
+set(gridDynMain_sources
```

Comment:
Capitalization? It's spelled this way in the manual

Reply:
ok